### PR TITLE
Feature/pages

### PR DESCRIPTION
**Pull request by SeanPed**

### CODE REVIEW

File: src/app/pages/InputPage/InputPage.module.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.body_div {
```
Comment (julialoeschel) [inline src/app/pages/InputPage/InputPage.module.css:1]: why did you move the overall body settings to only one page and call it body again? 
Maybe just call it ìnputPageContainer`or something not body-isch :)

File: src/app/pages/MainPage/MainPage.module.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
+.body_div {
```
Comment (julialoeschel) [inline src/app/pages/MainPage/MainPage.module.css:1]: same here - body (for me) is the overall setting - this might confuse people (me )  ;)

File: src/app/pages/MainPage/MainPage.module.css [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+.header_div {
+  grid-area: Top;
+  position: fixed;
+}
+.card_area_div {
+  border-top: 1px solid black;
+  grid-area: Bottom;
+  overflow-y: scroll;
+}
+.button_div {
```
Comment (julialoeschel) [inline src/app/pages/MainPage/MainPage.module.css:18]: is it necessary to call it all by its HTML-element? Calling it `header`, `card_area` or `button` / `buttonContainer` would make it easier.

File: src/app/App.tsx [hunk 4]
```diff
@@ -1,7 +1,17 @@
...
+import InputPage from './pages/InputPage/InputPage';
+import MainPage from './pages/MainPage/MainPage';
 
 function App(): JSX.Element {
-  return <h1>Hello World!</h1>;
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={<MainPage />}></Route>
+        <Route path="/input" element={<InputPage />}></Route>
```
Comment (F-Kirchhoff) [inline src/app/App.tsx:11]:
```suggestion
        <Route path="/" element={<MainPage />} />
        <Route path="/input" element={<InputPage />} />
```

File: src/app/pages/MainPage/MainPage.tsx [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+export default function MainPage(): JSX.Element {
+  return (
+    <div className={classes.body_div}>
+      <div className={classes.header_div}>
+        <h1>Hello World</h1>
+      </div>
+      <div className={classes.card_area_div}>
+        <div className={classes.button_div}>
+          <AddButton />
+        </div>
```
Comment (julialoeschel) [inline src/app/pages/MainPage/MainPage.tsx:16]: as long a syou dont style it, i think you dont need it, right?

**Review by julialoeschel — COMMENTED** [review]
great! 
just some small naming thoughts from my side!

File: src/app/components/AddButton/AddButton.tsx [hunk 6]
```diff
@@ -5,7 +5,7 @@ import { GiLightBackpack } from 'react-icons/gi';
 export default function AddButton(): JSX.Element {
   return (
     <button className={classes.add_button}>
-      <GiLightBackpack size={28} className={classes.add_button_symbol} />
+      <GiLightBackpack size={42} className={classes.add_button_symbol} />
     </button>
   );
 }
```
Comment (F-Kirchhoff) [inline src/app/components/AddButton/AddButton.tsx:11]: do you need a single component here?

File: src/app/pages/InputPage/InputPage.module.css [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 20% auto 20%;
+  justify-items: center;
+  size: 100vh;
+  grid-template-areas: 'Top' 'Center' 'Bottom';
```
Comment (F-Kirchhoff) [inline src/app/pages/InputPage/InputPage.module.css:7]: nice!

File: src/app/pages/InputPage/InputPage.tsx [hunk 8]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Form from '../../components/Form/Form';
+import classes from './InputPage.module.css';
+
+export default function InputPage(): JSX.Element {
+  return (
+    <div className={classes.body_div}>
+      <div className={classes.placeholder_div}></div>
+      <Form />
+      <div className={classes.placeholder_div}></div>
```
Comment (F-Kirchhoff) [inline src/app/pages/InputPage/InputPage.tsx:10]: I would solve this with grid row start and grid row end

File: src/app/pages/MainPage/MainPage.tsx [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+        <Card />
+        <Card />
+        <Card />
+        <Card />
+        <Card />
```
Comment (F-Kirchhoff) [inline src/app/pages/MainPage/MainPage.tsx:21]: use map instead

**Review by F-Kirchhoff — COMMENTED** [review]
lgtm :rabbit:
